src/components/Divider/Divider.tsx

```diff
@@ -46,8 +47,11 @@ class Divider extends UIComponent<Extendable<IDividerProps>, any> {
     /** Size multiplier (default 0) * */
     size: PropTypes.number,
 
-    /** A Divider can be formatted to show different levels of emphasis. */
-    type: PropTypes.oneOf(['primary', 'secondary']),
+    /** A Divider can appear to show a primary level of emphasis. */
+    primary: PropTypes.bool,
```

Comment:
If we even decide to go with introducing these props, we should make sure that those are not going to be used together. In order to achieve this, please use the `customPropTypes.disallow` in the definition of the properties accordingly.

Reply:
this would be definitely a good step, but would like to make it clear - it won't result in 'disallowing these props to be used together', but will just introduce a error message printed to console in case if this will happen. The problem is that, actually, at the API level we _cannot_ prevent these two to be specified at the same time.